jobs/dtd-catalog-total-load-exporter/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "dtd-catalog-total-load-exporter",
+  "main": "./dist/index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\"",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --watch --no-warnings --loader ts-node/esm ./src/index.ts",
+    "build": "tsc"
+  },
+  "devDependencies": {
+    "@types/lodash": "4.14.196",
+    "@types/node": "20.4.9",
+    "@typescript-eslint/eslint-plugin": "6.3.0",
+    "@typescript-eslint/parser": "6.3.0",
+    "eslint": "8.46.0",
+    "prettier": "^3.0.1",
+    "ts-node": "^10.9.1",
+    "typescript": "5.1.6",
+    "vitest": "0.34.1"
+  },
+  "dependencies": {
+    "@aws-sdk/client-s3": "^3.387.0",
+    "axios": "^1.4.0",
+    "chalk": "^5.3.0",
```

Comment:
we should probably remove `chalk` from the template script 😄

Reply:
Yup! Done!